config/initializers/jbuilder_prettify.rb

```diff
@@ -0,0 +1,26 @@
+require "jbuilder"
+
+class Jbuilder
+  ##
+  # Allows you to set @prettify manually in your .jbuilder files.
+  # Example:
+  #   json.prettify true
+  #   json.prettify false
+  #
+  attr_accessor :prettify
+
+  alias_method :_original_target, :target!
```

Comment:
__[rubocop]__
* Style/Alias: Use `alias` instead of `alias_method` in a class body. [:ref](https://github.com/bbatsov/ruby-style-guide#alias-method)

<!-- Comment by [SideCI](https://sideci.com) -->